presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -140,6 +144,7 @@ class SystemConfig : public ConfigBase {
   static constexpr bool kUseMmapAllocatorDefault{true};
   static constexpr bool kHttpEnableAccessLogDefault = false;
   static constexpr bool kHttpEnableStatsFilterDefault = false;
+  static constexpr uint64_t kHttpMaxAllocateBytesDefault = 64 << 10;
```

Comment:
Can we move the declaration closer to other declarations with similar type?

Reply:
This counter is http related so put them together.